app/src/test/java/tttTDD/Java/GameTest.java

```diff
@@ -35,14 +35,12 @@ public void PlayerShouldUpdateBoardWithinTheGame3x3() {
     @Test
     public void PerformTurnShouldReturnFalseWhenASpaceIsTaken() {
         game = new Game(gameConfig);
-        int humanInput1 = 0;
-        int computerInput1 = 1;
-        int humanInput2 = 0;
+        int[] humanMoves = {0, 1, 0};
+        // Moves played by computer => 1
```

Comment:
I generally don't like comments, but you did a good trade-off here. You needed to improve the usability of tests and lost some information, the important information came back as a comment. I would have done the same, then in the future, if you see an opportunity to pull the info back out of comments and get it into the code again it's always worth the refactor effort. :)